### PR TITLE
Mykhailo

### PR DESCRIPTION
**Pull request by tsmichael**
Try to merge WishList and Currency

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -99,10 +99,11 @@ hs_err_pid*
 # Ignore all .txt files in the directory doc/
 # doc/**/*.txt
 
-# Ignore .doc .pfd .docx
+# Ignore .doc .pfd .docx .json
```
Comment (ostret) [inline .gitignore]: not ideal 
it's not great to have jsons in root dir, but potentially you can use those resources

File: OpencartTest/Mykhailo.xml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+        <parameter name="myAccountDropdownText" value="Login"/>
+
+        <parameter name="fromSearchPageProductName" value="iPod Nano"/>
+        <parameter name="fromHomePageProductName" value="iPhone"/>
+        <parameter name="fromProductPageProductName" value="Samsung Galaxy Tab 10.1"/>
+
+        <classes>
+            <class name="test.TestWishList">
+                <methods>
```
Comment (ostret) [inline OpencartTest/Mykhailo.xml]: keep your xmls clean and don't comment out stuff
general suggestion - include whole classes, not methods 
you'll lose something important that way

File: OpencartTest/src/com/opencart/pages/HomePage.java [hunk 3]
```diff
@@ -1,13 +1,32 @@
...
+import java.util.ArrayList;
+import java.util.List;
 
 public class HomePage extends AbstractPageWithHeader {
 
+    private List<ProductContainersComponent> productContainersComponents;
+
     // CompareButtonMainPage
-   // "//a[text()='MacBook']/../../following-sibling::div/button[@data-original-title='Add to Wish List']"
+    // "//a[text()='MacBook']/../../following-sibling::div/button[@data-original-title='Add to Wish List']"
```
Comment (ostret) [inline OpencartTest/src/com/opencart/pages/HomePage.java]: what's gonna happen if macbook product is deleted ?

File: OpencartTest/src/com/opencart/tools/JsonDataConfig.java [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+import java.io.FileReader;
+import java.io.IOException;
+
+public class JsonDataConfig {
+
+
+    private FileReader reader;
+    private JSONArray userLoginArray;
+
+    public JsonDataConfig(String fileName) {
```
Comment (ostret) [inline OpencartTest/src/com/opencart/tools/JsonDataConfig.java]: look into jackson json serialization / desirialization . just generally easier to work with
refactor to have filereader and parser separately

File: OpencartTest/src/com/opencart/pages/HomePage.java (hunk 3 above)
Comment (tsmichael) [reply OpencartTest/src/com/opencart/pages/HomePage.java]: it was already commented  - so this trash code i removed

File: OpencartTest/src/com/opencart/tools/JsonDataConfig.java (hunk 4 above)
Comment (tsmichael) [reply OpencartTest/src/com/opencart/tools/JsonDataConfig.java]: Can i make json changing in future merge?

Comment (tsmichael) [reply OpencartTest/src/com/opencart/tools/JsonDataConfig.java]: @ostret Can i make json changing in future merge?

**Pull request merged**